popup.vim

```diff
@@ -134,6 +138,21 @@ else
   endfunction
 endif
 
+"
+" _list
+"
+if has('nvim')
+  function! s:_list() abort
+    let l:info_list = getwininfo()
+    let l:winid_list = map(info_list, 'v:val["winid"]')
+    return filter(winid_list, 's:_is_popup_window(v:val)')
+  endfunction
+else
+  function! s:_list() abort
+    return popup_list()
```

Comment:
It should be `return filter(popup_list(), 'popup_getpos(v:val).visible')`, I think.